tree/ntuple/v7/src/RField.cxx

```diff
@@ -447,6 +523,26 @@ void ROOT::Experimental::Detail::RFieldBase::ReadGlobalImpl(ROOT::Experimental::
    R__ASSERT(false);
 }
 
+std::size_t ROOT::Experimental::Detail::RFieldBase::ReadBulkImpl(const RBulkSpec &bulkSpec)
+{
+   const auto valueSize = GetValueSize();
+   std::size_t nRead = 0;
+   for (std::size_t i = 0; i < bulkSpec.fCount; ++i) {
+      // Value not needed
+      if (!bulkSpec.fMaskReq[i])
+         continue;
+
+      // Value already present
+      if (bulkSpec.fMaskAvail[i])
+         continue;
+
+      Read(bulkSpec.fFirstIndex + i, reinterpret_cast<unsigned char *>(bulkSpec.fValues) + i * valueSize);
```

Comment:
RBulkSpec stores `fValues` as `void *` while `RBulk` uses `unsigned char *`. I don't have an opinion either way but it would be less surprising to be consistent.